wasabi/tests/test_printer.py

```diff
@@ -42,7 +42,7 @@ def test_printer():
 
 
 def test_printer_print():
-    p = Printer()
+    p = Printer(timestamp=True)
```

Comment:
If we wanted to have a proper test for this, we could do something like this:

```python
p = Printer(no_print=True, timestamp=True)
result = p.text("Hello world")
```

And then check `result` against a regex to make sure that it's prefixed by a timestamp (any timestamp).

Reply:
Done! I kept the regex simpel and broad, for testing purposes I think that's fine.